src/utils/articles_api.js

```diff
@@ -4,8 +4,10 @@ const articlesAPI = axios.create({
   baseURL: "https://news-api-vzif.onrender.com/api/articles",
 });
 
-export const getArticleSummaries = () => {
-  return articlesAPI.get("/").then((response) => {
+export const getArticleSummaries = (topic) => {
+  let queryString = "/";
+  if (topic) queryString += `?topic=${topic}`;
```

Comment:
We don't need to make this so complicated. Axios allows you to pass the path to the http action as well as params. Take a look here https://notes.northcoders.com/courses/js-front-end/axios#axios-instance

Reply:
This is going to make my life much easier in the next step, thanks for pointing me in the right direction 🙂